testing/jormungandr-integration-tests/src/common/jormungandr/process.rs

```diff
@@ -54,6 +67,60 @@ impl JormungandrProcess {
         }
     }
 
+    pub fn status(&self, strategy: &StartupVerificationMode) -> Status {
+        let port_occupied_msgs = ["error 87", "error 98", "panicked at 'Box<Any>'"];
```

Comment:
The match on "panicked at 'Box<Any>'" may produce false positives. It's better to fix the diagnostic in jormungandr to provide a more specific line to match on.

Reply:
agree, I would rather fix this on another PR though, this comes from previous implementation.